### PR TITLE
Fix part of #973 : Fix app module tests for Roboelectric in PinPasswordActivityTest

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #973
Fix for PinPasswordActivityTest

### Espresso 
Screenshot 2020-09-17 at 18 59 41

### Robolectric 
Screenshot 2020-09-17 at 18 54 37

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/profile/PinPasswordActivityTest.kt [hunk 1]
```diff
@@ -709,6 +705,31 @@ class PinPasswordActivityTest {
...
+    val typeTextViewAction = typeText(text)
+    return object : ViewAction {
+      override fun getDescription(): String = typeTextViewAction.description
+
+      override fun getConstraints(): Matcher<View> = typeTextViewAction.constraints
+
+      override fun perform(uiController: UiController?, view: View?) {
+        // Appending text only works on Robolectric, whereas Espresso needs to use typeText().
+        if (Build.FINGERPRINT.contains("robolectric", ignoreCase = true)) {
+          (view as? EditText)?.append(text)
```
Comment (aggarwalpulkit596) [inline app/src/sharedTest/java/org/oppia/app/profile/PinPasswordActivityTest.kt:724]: this indentation looks incorrect to me?

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/app/profile/PinPasswordActivityTest.kt:724]: no, I guess. I had already done the reformat thing.

Comment (aggarwalpulkit596) [reply app/src/sharedTest/java/org/oppia/app/profile/PinPasswordActivityTest.kt:724]: Okay thanks

**Review by aggarwalpulkit596 — APPROVED** [review]
LGTM

**Pull request merged**